ptz-pelco_p.hpp

```diff
@@ -0,0 +1,55 @@
+/* Pan Tilt Zoom camera pelco-p instance
+ *
+ * Copyright 2021 Luuk Verhagen <developer@verhagenluuk.nl>
+ *
+ * SPDX-License-Identifier: GPLv2
+ */
+
+#pragma once
+#include "ptz-device.hpp"
+#include "UART.hpp"
+#include <util/base.h>
+
+class PelcoPCmd {
+public:
+	unsigned char cmd[4];
+
+	PelcoPCmd(const unsigned char data_1, const unsigned char data_2, const unsigned char data_3, const unsigned char data_4);
+};
```

Comment:
As discussed, this doesn't do anything special. Just use a QByteArray directly.

With VISCA I created a new class because I created encoder/decoder helper classes to make the protocol more extensible. At this stage you're not trying to do that, so adding a class just makes the code more complex.